### ISSUE TITLE
File necessary for beta2bam missing

### ISSUE THREAD

**Issue by rowanhowellGE** [nloyfer/wgbs_tools#16]
Hi,

I am trying to convert a beta file to a bam using beta2bam. I have setup hg38 as the reference. When i run: 
`wgbstools beta2bed GSM5652219_Oligodendrocytes-Z000000TK.hg38.beta -o GSM5652219_Oligodendrocytes-Z000000TK.hg38.tsv`
I get the following error:
```
Invalid input argument
Invalid reference path: /wgbs_tools/references/hg38/CpG.bed.gz
```
When I look in the "/wgbs_tools/references/hg38" folder, the only entry is hg38.fa.gz. What steps are required to generate the CpG.bed.gz file?

Thanks

**Comment by yonniejon** [comment]
Hi, did you run the init_genome command?

`wgbstools init_genome hg38`

This downloads necessary annotation files and creates all other necessary reference files including CpG.bed.gz

**Comment by rowanhowellGE** [comment]
Hi,
I did run init_genome, hence why the hg38.fa.gz file was created. However, looking more closely the init_genome command did raise an error `samtools: error while loading shared libraries: libncurses.so.5: cannot open shared object file: No such file or directory` . This was fixed by downloading the libncurses5 package and I now have CpG.bed.gz. Maybe libncurses5 should be added to dependencies?
